frontend/lib/views/truck_detail.dart

```diff
@@ -128,6 +133,18 @@ class _TruckDetailState extends State<TruckDetailView> {
         : Container();
   }
 
+  Widget _truckRating(){
+    return Row(
+      mainAxisSize: MainAxisSize.min,
+      children: List.generate(5, (index) {
```

Comment:
Since we don't have api for the rating yet, I think we can remove it for now? In the future when the api is implemented, we could add this back to the page.